_pkgdown.yml

```diff
@@ -13,45 +13,112 @@ reference:
   contents:
   - wb_to_df
   - write_xlsx
-  - wbWorkbook
   - wb_workbook
   - wb_load
+  - wbWorkbook
+  - wb_open
   - wb_save
 
-- title: wb add and remove functions
+- title: Main functions to interact with the contents of a Workbook object
+- subtitle: Add content to a worksheet
+  desc: >
+    These functions help you write in worksheets. They invisibly return the
+    `wbWorkbook` object, except for `get_` functions, who return a character vector,
+    unless specified otherwise.
+  contents:
+  - wb_add_worksheet
```

Comment:
shouldn't this list be sorted alphabetically? it's small enough that this would make sense

Reply:
I tried sorting it by order and frequency of use. Don't mind doing alphabetical. I will open a (hopefully) last doc-related PR, where I will finalize using `#' @family` to get rid of most of this